src/main/java/gregtech/api/capability/impl/miner/MinerLogic.java

```diff
@@ -194,6 +194,7 @@ public void performMining() {
                 this.isDone = true;
                 this.wasActiveAndNeedsUpdate = true;
                 this.setActive(false);
+                this.setWorkingEnabled(false);
```

Comment:
Doesn't this mean that the miner will be in a paused state when it cannot find blocks to mine? Shouldn't it stay in an active but not mining state instead? If so, I think this line should be removed.

Reply:
It does, but i haven't noticed any issues with that. I guess i could move it to resetArea(), so it doesn't pause until you start changing the area.